troubleshooting/known_issues/console_log_storage_filling.md

```diff
@@ -0,0 +1,65 @@
+# Console logs fill all available storage space.
+The log rotation functionality of the console logging services have a bug where
+the files are not rotated correctly.  This will lead to the logs continuing to
+expand until eventually all space is consumed on the storage. This can lead to
+issues where the log files cease to capture log output and can sometimes lead
+to the `cray-console-node` pods may not correctly connect to nodes.
+
+To see if this is an issue, check the used space on the /var/log/ mount in the `cray-console-node` pods:
```

Comment:
1. Verify if the issue exists by checking the used space on the `/var/log/` mount in the `cray-console-node` pods:

   ```text
   ncn# kubectl -n services exec -it cray-console-node-0 -- sh
   sh-4.4$ df -h | grep var
   Filesystem                     Size  Used Avail Use% Mounted on
   /volumes/csi/csi-vol-ab81e37b  100G   99G  100G  99% /var/log
   ```